contribs/gmf/apps/desktop/js/controller.js

```diff
@@ -108,3 +110,15 @@ goog.inherits(app.DesktopController, gmf.AbstractDesktopController);
 
 
 app.module.controller('DesktopController', app.DesktopController);
+
+app.module.decorator('gmfContextualdatacontentDirective',
+    ['$delegate', 'contextualdataTemplateUrl',
```

Comment:
That looks a bit weird but I couldn't find any other way to prevent an error with 'gmfContextualdatacontentDirective' not being available when built. If someone can try to dig into this and find a more elegant to fix this, I'd be happy.